cmd/up.go

```diff
@@ -375,10 +382,13 @@ func newUpCmd() *cobra.Command {
 		&message, "message", "m", "",
 		"Optional message to associate with the update operation")
 
-	targets = cmd.PersistentFlags().StringArrayP(
-		"target", "t", []string{},
+	cmd.PersistentFlags().StringArrayVarP(
+		&targets, "target", "t", []string{},
 		"Specify a single resource URN to update. Other resources will not be updated."+
-			" Multiple resources can be specified using: --target urn1 --target urn2")
+			" Multiple resources can be specified using --target urn1 --target urn2")
+	cmd.PersistentFlags().StringArrayVar(
+		&replaces, "replace", []string{},
+		"Specify resources to replace. Multiple resources can be specified using --replace run1 --replace urn2")
```

Comment:
Am I right that `pulumi up --replace urn` does a normal update (of all resources), but forces `urn` to replace even if it wouldn't change?

So to do a targeted replace - you would do `pulumi up --target urn --replace urn`?  And to just do a normal update that forces a VM to replace, you would do `pulumi up --replace urn`?

Reply:
That's correct.